proto/opencdc/v1/opencdc.proto

```diff
@@ -13,8 +13,12 @@ option (metadata_created_at) = "opencdc.createdAt";
 option (metadata_read_at) = "opencdc.readAt";
 option (metadata_version) = "opencdc.version";
 option (metadata_collection) = "opencdc.collection";
-option (metadata_schema_id) = "opencdc.schema.id";
-option (metadata_schema_type) = "opencdc.schema.type";
+option (metadata_key_schema_name) = "opencdc.key.schema.name";
+option (metadata_key_schema_version) = "opencdc.key.schema.version";
+option (metadata_key_schema_type) = "opencdc.key.schema.type";
```

Comment:
Hmm, why do we need the type in the record metadata? 🤔  Seems like we can get away without this.

Reply:
@lovromazgon I don't seem to see why it could be needed. I'll remove for now and will add back if I find the reason.

Reply:
@lovromazgon something tells me it is needed indeed since for both key and payload we plan go use name, version and type.

Reply:
My comment was based on the fact that we only need the [subject and version](https://github.com/ConduitIO/conduit-connector-protocol/blob/d24ebd2b15728e19b841b4e94f063c76d93fce5f/proto/conduit/v1/schema.proto#L35-L36) to fetch a schema, so it seems like type is a bit of metadata that we don't need and actually can't do anything with it. Let's say you see a record and it says the payload schema type is "avro" - it doesn't give you much information, since you need the actual schema to do anything with the encoded bytes.

By the way - we should rename the key to `opencdc.key.schema.subject`, to be consistent with the rest of the code.

Reply:
@lovromazgon got it, that makes sense to me. I'll change.